### ISSUE TITLE
Нельзя скрыть вкладку "Настройки" через "Настройку форм"

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modx-pro/miniShop2#476]
## Сообщение об ошибке
Нельзя скрыть вкладку "Настройки" через "Настройку форм":
- Во-первых, всегда показывается чекбокс "Родительский ресурс" в ресурсе, даже если чекбокс отключен в "Настройке форм".
- Во-вторых, нет чекбокса для "Показывать в дереве" в "Настройке форм".

Возможно из-за этого и нельзя скрыть всю вкладку через отключение области `modx-page-settings`.

![miniShop_settings](https://user-images.githubusercontent.com/12523676/99068222-1e294580-25bd-11eb-83dc-f842b0b0c44f.png)

### Шаг для воспроизведения
Попробуйте скрыть область `modx-page-settings`.

### Наблюдаемое поведение
Нельзя скрыть всю область `modx-page-settings`.
Нет чекбокса "Показывать в дереве", а чекбокс "Родительский ресурс" не скрывается, даже если чекбокс отключен.

### Ожидаемое поведение
Все чекбоксы должны присутствовать и быть отключаемыми. Так же как и любая область.

### Среда
miniShop2 2.5 >

**Comment by Ibochkarev** [comment]
@Ruslan-Aleev https://github.com/Ibochkarev/miniShop2/pull/501 - это не исправляет?

**Comment by Ruslan-Aleev** [comment]
Нет, тут с настройкой форм дело, причем баг давнишний, я на старых версиях смотрел.
